package.json

```diff
@@ -1989,6 +1990,11 @@
                 "title": "%vscode-docker.commands.configure%",
                 "category": "%vscode-docker.commands.category.docker%"
             },
+            {
+                "command": "vscode-docker.configureDocker",
```

Comment:
I think we should name it `vscode-docker.configureCompose`. The existing `vscode-docker.configure` is already too vague of a name but not something we can change, let's make this one more specific.

Reply:
Not sure how I came up with ConfigureDocker while the functions are named configureCompose